nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/GetHDFSFileInfo.java

```diff
@@ -326,6 +326,13 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             getLogger().error("Interrupted while performing listing of HDFS", e);
             ff = session.putAttribute(ff, "hdfs.status", "Failed due to: " + e);
             session.transfer(ff, REL_FAILURE);
+        } catch (final IOException e) {
```

Comment:
The "Interrupted" log message copied from the `InterruptedException` clause is suboptimal.

In general instead of
```java
        } catch (final IOException e) {
            // Catch GSSExceptions and reset the resources
            if (!handleAuthErrors(e, session, context)) {
                getLogger().error("Interrupted while performing listing of HDFS", e);
                ff = session.putAttribute(ff, "hdfs.status", "Failed due to: " + e);
                session.transfer(ff, REL_FAILURE);
            }
        } catch (final Exception e) {
            getLogger().error("Failed to perform listing of HDFS due to {}", new Object[]{e});
            ff = session.putAttribute(ff, "hdfs.status", "Failed due to: " + e);
            session.transfer(ff, REL_FAILURE);
        }
```
we could simply have
```java
        } catch (final Exception e) {
            if (!handleAuthErrors(e, session, context)) {
                getLogger().error("Failed to perform listing of HDFS due to {}", new Object[]{e});
                ff = session.putAttribute(ff, "hdfs.status", "Failed due to: " + e);
                session.transfer(ff, REL_FAILURE);
            }
        }
```